test/xref2/with.t/run.t

```diff
@@ -0,0 +1,48 @@
+This test checks that when we have expressions of the form `sig ... end with ...` that we replace
+them with simple `sig ... end` - any types or modules referred to in the `with` clauses will not
+be able to link anywhere because we don't render the inner signature anywhere.
+
+  $ cat test.ml
+  module type S = sig
+    type t
+    type u
+  end with type t = int with type u = int
+  
+
+  $ cat compile.sh
```

Comment:
It is possible to write that file from here, so it is written only once:
```
  $ cat > compile.sh <<EOF
  #!/bin/bash
  
  ocamlc -c -bin-annot test.ml
  odoc compile page.mld --child module-test
  odoc compile -I . --parent page test.cmt
  odoc link -I . test.odoc
  odoc html-generate --indent -o html test.odocl
  odoc support-files -o html
  EOF
```

Reply:
true - though I find it convenient to have the script handy for testing - I often seem to find it tricky to locate the dir in which dune has executed the test!